proto/api/v1/network_graph_service.proto

```diff
@@ -71,6 +71,15 @@ message GetExternalNetworkEntitiesResponse {
   repeated storage.NetworkEntity entities = 1;
 }
 
+message GetExternalNetworkFlowsRequest {
+  string cluster_id = 1;
+  string deployment_id = 3;
```

Comment:
Why is deployment_id not 2? This feels very odd, I would understand it on an existing message, but this one looks brand new.

Reply:
I was sure I'd fixed that - apologies. There used to be a `query` parameter, but it was unused due to the changes I made to the flow store, so I dropped it. I'll fix this.